Cargo.toml

```diff
@@ -26,8 +26,8 @@ chrono = { version = "0.4", default-features = false }
 uuid = { version = "1.3", default-features = false }
 duckdb = { version = "0.8.1", features = ["bundled"] }
 
-fluvio = { git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.12" }
-fluvio-connector-common = { features = ["derive"], git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.12" }
+fluvio = { git = "https://github.com/infinyon/fluvio.git", rev = "d9a76327b674614c7373a9c12eb6a0ed2fcc4dc0" }
```

Comment:
Shouldn't this be using a tag?

Reply:
We have a tag when we bump fluvio to a release version, but if we want to release the connector in between fluvio releases, we use a commit.